skyve-war/src/main/java/modules/admin/ChangePassword/ChangePasswordNotificationJob.java

```diff
@@ -0,0 +1,61 @@
+package modules.admin.ChangePassword;
+
+import java.util.List;
+
+import org.skyve.EXT;
+import org.skyve.impl.util.UtilImpl;
+import org.skyve.job.Job;
+import org.skyve.util.Mail;
+import org.skyve.util.Util;
+
+import modules.admin.domain.Contact;
+import modules.admin.domain.User;
+
+/**
+ * This job sends a notification email to the current user, confirming that they just changed their password.
+ * 
+ * @author Simeon Solomou
+ */
+public class ChangePasswordNotificationJob extends Job {
+
+	private static final String EMAIL_SUBJECT = "Security alert: password change";
+	private static final String EMAIL_BODY = "Hello %s,"
+			+ "<br/><br/>"
+			+ "This is a courtesy email to let you know that your password has been changed as requested."
+			+ "<br/><br/>"
+			+ "If you have any concerns regarding this, please contact us <a href=\"mailto: info@bizhub.com.au\">here</a>.";
```

Comment:
This should not be hard coded. Biz Hub do not want to be notified if someone else is responsible for the application. Use `Util.getSupportEmailAddress()` instead.